### PR TITLE
Reblocking warning on ExomeGermlineSingleSample.wdl

### PR DESCRIPTION
**Pull request by ldgauthier**
A user got burned, so I'm putting a warning directly in the WDL

### CODE REVIEW

**Review by jessicaway — APPROVED** [review]
Thanks @ldgauthier!

File: pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.wdl [hunk 1]
```diff
@@ -6,6 +6,10 @@ version 1.0
 ## generation) according to the GATK Best Practices (June 2016) for germline SNP and
 ## Indel discovery in human exome sequencing data.
 ##
+## NOTE: "Reblocking" of the output GVCF, which reduces hom-ref genotype quality resolution
```
Comment (samuelklee) [inline pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.wdl:9]: Is there a comma missing after `resolution`?

And perhaps a bit of tweaking: `"Reblocking" of the output GVCF is now done by default. This reduces the genomic resolution of hom-ref genotype qualities, which may affect de novo calling in trios.` Or something along those lines. (Note also "genomic resolution," which may prevent confusion with decreased resolution of GQ bins or other tangentially related concepts.)

**Review by samuelklee — APPROVED** [review]
Just some minor grammatical suggestions, otherwise LGTM!

**Comment by ldgauthier** [comment]
Augh, I always forget about the changelog.  Yes @jessicaway that would be lovely, thanks!

**Comment by jessicaway** [comment]
test this please

**Comment by jessicaway** [comment]
add to whitelist

**Pull request merged**